### ISSUE TITLE
Virtualbox Cask does not work but same file works when manually installed

### ISSUE THREAD

**Issue by harshalbhatia** [Homebrew/homebrew-cask#89138]
#### General troubleshooting steps

- [x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [x] I have retried my command with `--force`.
- [x] I ran `brew update-reset && brew update` and retried my command.
- [x] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [x] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [x] I made doubly sure this is not a [checksum does not match](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md) error.

#### Description of issue

Installing `virtualbox` cask through brew cask worked. However, on trying to boot into a VM, it crashes. Attaching screenshot. I thought this was an issue with virtualbox itself, but i uninstalled the cask and retried to install it using the same file (used it from Homebrew cache) and it successfully worked. Had a look at security also and gave Virtualbox permissions wherever i could find the option, but it still doesn't work.

![image](https://user-images.githubusercontent.com/7516211/92993741-ba799200-f511-11ea-9c3c-82307209ec46.png)

#### Command that failed

```
Not a command
```

#### Output of command with `--force --verbose --debug`

```
NA
```

#### Output of `brew doctor --verbose`

```
==> Cask Environment Variables:
BUNDLE_PATH
CHRUBY_VERSION
GEM_HOME
GEM_PATH
HOMEBREW_CASK_OPTS
LC_ALL
PATH
RBENV_VERSION
RUBYLIB
RUBYOPT
RUBYPATH
SHELL
==> $LOAD_PATHS
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ruby-macho-2.2.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-rspec-1.43.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-performance-1.8.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/plist-3.5.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/patchelf-1.3.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mechanize-2.7.6/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/elftools-1.1.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/bindata-2.4.8/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/activesupport-6.0.3.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/zeitwerk-2.4.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/tzinfo-1.2.7/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/thread_safe-0.3.6/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/i18n-1.8.5/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/concurrent-ruby-1.1.7/lib/concurrent-ruby
/Library/Ruby/Gems/2.6.0/gems/json-2.2.0/lib
/Library/Ruby/Gems/2.6.0/extensions/universal-darwin-19/2.6.0/json-2.2.0
/Library/Ruby/Site/2.6.0
/Library/Ruby/Site/2.6.0/universal-darwin19
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby/2.6.0
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby/2.6.0/universal-darwin19
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/universal-darwin19
/usr/local/Homebrew/Library/Homebrew
==> Homebrew Version
2.5.0-119-g3ecbd91
==> macOS
10.15.6
==> SIP
Enabled
==> Java
14.0.1
==> Homebrew Cask Staging Location
/usr/local/Caskroom
==> Homebrew Cask Taps:
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask (3657 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask-versions (160 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask-fonts (1561 casks)
/usr/local/Homebrew/Library/Taps/kernel/homebrew-kernel (2 casks)
```

#### Output of `brew tap`

```
gojek/gojek
homebrew/cask
homebrew/cask-fonts
homebrew/cask-versions
homebrew/core
homebrew/services
kernel/kernel
kong/kong
mongodb/brew
```

**Comment by vitorgalvao** [comment]
After we install it, it’s out of our hands. Because we install with standard tools that come with macOS (`installer`), for this to happen it would need to either be a bug in `installer`, Virtualbox, or your setup. I’ll leave open for a while to see if someone else has the issue, but the cask is very popular and no one else has reported this, so it’s bound to be something in your particular setup.

```
virtualbox
30 days: 9,691 (#11)
90 days: 28,278 (#10)
365 days: 132,027 (#12)
Age: 2862 days (added 2012, November 11)
```

**Comment by smcgivern** [comment]
I'm also seeing this @harshalbhatia, but I agree with @vitorgalvao - this is not a Homebrew problem.

In my case, the macOS crash log showed (in thread 2) a line about `__CRASHING_DUE_TO_PRIVACY_VIOLATION__`. Googling that got me to a bunch of posts on the Virtualbox forums about this, like https://forums.virtualbox.org/viewtopic.php?f=8&t=99299.

I was able to see the device requests by launching `virtualbox` through zsh (not bash, curiously). It was asking for microphone access, and my VM doesn't need that anyway, so I just disabled audio in the VM's settings.

If yours isn't a similar case then it would probably be helpful to post the crash log you get from macOS to the Virtualbox forums.
